README.md

```diff
@@ -32,6 +32,9 @@ Currently the panes available include:
 
 The solid-app-set panes are built using a set of widgets and utilities in [https://github.com/linkeddata/solid-ui](https://github.com/linkeddata/solid-ui)
 
+To help onboarding for new users we're using [user types](./Documentation/user-types.md) to limit the number of panes that is presented
```

Comment:
Heh, great job on using "number ... is presented" rather than "number ... are presented" ;)